.github/workflows/ci.yml

```diff
@@ -0,0 +1,318 @@
+      run: mkdir dependencies
+
+    - name: Cache OCCT
+      id: cache-occt
+      uses: actions/cache@v2
+      with:
+        path: dependencies/occt_install
+        key: occt-b-V7_6_0-${{matrix.os}}
+
+    - name: Checkout OCCT
+      if: steps.cache-occt.outputs.cache-hit != 'true'
+      uses: actions/checkout@v2
+      with:
+        repository: Open-Cascade-SAS/OCCT
+        submodules: 'true'
+        path: './dependencies/occt'
+        ref: V7_6_0
+
+    - name: Setup OCCT
+      if: steps.cache-occt.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}/dependencies
+      run: |
+        mkdir occt_build
+        mkdir occt_install
+
+    - name: Configure OCCT
+      if: steps.cache-occt.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}/dependencies/occt_build
+      run: >
+        cmake ../occt
+        -DBUILD_LIBRARY_TYPE=Static
+        -DBUILD_ADDITIONAL_TOOLKITS="TKSTEP;TKIGES;TKMesh"
+        -DBUILD_MODULE_ApplicationFramework=OFF
+        -DBUILD_MODULE_DataExchange=OFF
+        -DBUILD_MODULE_Draw=OFF
+        -DBUILD_MODULE_FoundationClasses=OFF
+        -DBUILD_MODULE_ModelingAlgorithms=OFF
+        -DBUILD_MODULE_ModelingData=OFF
+        -DBUILD_MODULE_Visualization=OFF
+        -DCMAKE_BUILD_TYPE=Release
+        -DCMAKE_INSTALL_PREFIX:PATH=../occt_install
+        ${{ matrix.os == 'windows-latest' && '-Ax64 -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded' || null }}
+
+    - name: Build OCCT
+      if: steps.cache-occt.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}/dependencies/occt_build
+      run: cmake --build . --parallel 2 --target install --config Release
+
+    - name: Patch OCCT install macos
+      if: |
+        steps.cache-occt.outputs.cache-hit != 'true' &&
+        matrix.os == 'macos-latest'
+      working-directory: ${{github.workspace}}/dependencies
+      run: sed -i '' 's/OpenCASCADE_LIBRARIES [A-Za-z0-9;]\+/OpenCASCADE_LIBRARIES TKSTEP;TKSTEPAttr;TKSTEP209;TKSTEPBase;TKIGES;TKBool;TKMesh;TKXSBase;TKBO;TKPrim;TKShHealing;TKTopAlgo;TKBRep;TKGeomAlgo;TKGeomBase;TKG3d;TKG2d;TKMath;TKernel/' occt_install/lib/cmake/opencascade/OpenCASCADEConfig.cmake
+
+    - name: Patch OCCT install ubuntu
+      if: |
+        steps.cache-occt.outputs.cache-hit != 'true' &&
+        matrix.os == 'ubuntu-latest'
+      working-directory: ${{github.workspace}}/dependencies
+      run: sed -i 's/OpenCASCADE_LIBRARIES [A-Za-z0-9;]\+/OpenCASCADE_LIBRARIES TKSTEP;TKSTEPAttr;TKSTEP209;TKSTEPBase;TKIGES;TKBool;TKMesh;TKXSBase;TKBO;TKPrim;TKShHealing;TKTopAlgo;TKBRep;TKGeomAlgo;TKGeomBase;TKG3d;TKG2d;TKMath;TKernel/' occt_install/lib/cmake/opencascade/OpenCASCADEConfig.cmake
+
+    # TODO rename cache 25/11
+    - name: Cache VTK
+      id: cache-vtk
+      uses: actions/cache@v2
+      with:
+        path: dependencies/vtk_install
+        key: vtk-unversioned-${{matrix.vtk_version}}-${{matrix.os}}
```

Comment:
what means unversioned?

Reply:
the install of VTK does not contains the version string, like vtk9.0.1. I will remove this once the previous cache is removed by github (7days)